pandas/core/generic.py

```diff
@@ -6827,6 +6827,10 @@ def fillna(
         reindex : Conform object to new index.
         asfreq : Convert TimeSeries to specified frequency.
 
+        Notes
+        -----
+        For non-object dtype, ``value=None`` will use the NA value of the dtype.
```

Comment:
Could you link to https://pandas.pydata.org/docs/user_guide/missing_data.html#values-considered-missing?